src/demo/polymer-lint.ts

```diff
@@ -40,7 +40,7 @@ async function getWarnings(analyzer: Analyzer, localPath: string):
     Promise<Warning[]> {
       try {
         const document = await analyzer.analyze(localPath);
-        return document.getWarnings();
+        return document.getWarnings({lookInDependencies: false});
```

Comment:
bikeshed time: that's kind of a long option name. `followImports`, `imported`, `transitive`, `deep`? `deep` might be too short, but on the other hand we can document the difference between `deep` and `nested`.